packages/react-ui-core/src/Ratings/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as RatingBar } from './RatingBar'
```

Comment:
Can we also add the following export to match the way the other components are done?
```
export { default } from './RatingBar'
```